src/material-examples/cdk-virtual-scroll-context/cdk-virtual-scroll-context-example.ts

```diff
@@ -0,0 +1,12 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+
+/** @title Virtual scroll context variables */
+@Component({
+  selector: 'cdk-virtual-scroll-context-example',
+  styleUrls: ['cdk-virtual-scroll-context-example.css'],
+  templateUrl: 'cdk-virtual-scroll-context-example.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CdkVirtualScrollContextExample {
+  items = Array.from({length: 100000}).map((_, i) => `Item #${i}`);
```

Comment:
nit: Can be reduced to `new Array(100000).fill(0).map((_, i) => `Item #${i}`)` which is slightly shorter.

Reply:
Yeah I originally had that but switched it to this. This feels more readable to me, so I think its better for the purposes of an example.